### ISSUE TITLE
Create a website for Avocado

### ISSUE THREAD

**Issue by peter-evans** [AvocadoNetwork/governance#3]
Avocado needs a website to introduce the concept, history and how to get involved in the community.

### Tasks
-  Create a website for Avocado

### Detail
- GitHub Pages/Jekyll would probably be the best option to begin with as hosting it will be free.
- Basic content should include an About/History of Avocado, the latest news and updates, links to social media and the Avocado wallet interface, etc.
- It should be easy to add new pages and content using the same style/template.
- All graphics/media used should be self-made or royalty free.
- Placeholders for graphics to be created as a separate task may be used.

### Deliverables
- Fork [this repository](https://github.com/AvocadoNetwork/web) and submit a PR with the website code

### Reference
See the information here for basic content: https://github.com/AvocadoNetwork/governance/blob/master/README.md
Avocado icon:
https://github.com/AvocadoNetwork/media/tree/master/graphics/icon

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

__Work has been started__.

 1. [@vikaskyadav](https://gitcoin.co/profile/vikaskyadav) 

 has committed to working on this project to be completed 12 months from now.


@peter-evans, __please see the below comments / questions regarding approach for this ticket from the bounty hunter(s):__ 
- [@vikaskyadav](https://gitcoin.co/profile/vikaskyadav): We can work on Graphics together. will be needing help in that.  I have some ideas for  elegant looking landing pages.

**Comment by ctrlaltvikas** [comment]
@peter-evans , plz mail me on vikasharry03@gmail.com for details and how you would like the platform to be designed.

**Comment by peter-evans** [comment]
@vikaskyadav Thank you for starting this task!

Unless you need to share sensitive information like passwords, private keys, etc., I would like to keep discussion about the task here. It serves as a record of how decisions were made and allows other members of the community to contribute to the discussion.

As for how it should be designed. I think that to start with we would like to be able to host the website with GitHub Pages. We don't have any funds for hosting at the moment and GitHub Pages is free. I'm not very familiar with exactly what you can and can't do with GitHub Pages but there will of course be limitations.

**Comment by peter-evans** [comment]
@vikaskyadav I added Avocado's icon graphics here:
https://github.com/AvocadoNetwork/media/tree/master/graphics/icon

Let us know if you have any further questions or need help with anything.

**Comment by gitcoinbot** [comment]
@vikaskyadav Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475?snooze=1) | [3 days](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475?snooze=3) | [5 days](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475?snooze=5) | [10 days](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475?snooze=10) | [100 days](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475?snooze=100)

**Comment by ctrlaltvikas** [comment]
yeah. @peter has provided the relevant items. I can start working on the
app now.

**Comment by ctrlaltvikas** [comment]
We will be going with React.js + Next.js ( for static site generation) and we can, later on, deploy over GitHub pages or surge.sh.

**Comment by ctrlaltvikas** [comment]
@peter-evans  could you please enlist major Items which are to be kept on the navbar? Such as About, Latest News, Contributing to the project, etc?

**Comment by peter-evans** [comment]
@vikaskyadav It's difficult to be definite without seeing the design of the site but I guess to start with we should have "About," "Latest News," and maybe "Apps" or something like that. The "Apps" page would have links to the Avocado wallet and any future DApps we create. Maybe it would be a good idea to have a "Community" page where we have links to social media and how to get involved in the project in various ways.

The content of the site is likely to change over time so please try to make it easy for us to add/edit pages and their associated menu items.

**Comment by gitcoinbot** [comment]
@vikaskyadav Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475?snooze=1) | [3 days](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475?snooze=3) | [5 days](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475?snooze=5) | [10 days](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475?snooze=10) | [100 days](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475?snooze=100)

**Comment by peter-evans** [comment]
@vikaskyadav Are you still working on this? Let us know how it's going. Feel free to submit a few PRs if you want to do it in stages.

**Comment by ctrlaltvikas** [comment]
I think I messed up git history. Unable to submit a PR. You can check some
layout on http://governance.surge.sh/. Open for feedback.

**Comment by peter-evans** [comment]
Looks great. As long is it's fairly straightforward to update and add content I sure it will be fine. Looking forward to seeing how it's built.

When do you aim to be able to complete this task?

**Comment by ctrlaltvikas** [comment]
Yeah. Adding new items will be very easy with this. Do let me know some content that can be kept there.
I would like to complete this asap if we have got good clarity regarding the content to be kept.
Have developed this way that you won't need to worry about the SEO and other things.

**Comment by peter-evans** [comment]
There is basic content available here that you can use:
https://github.com/AvocadoNetwork/governance/blob/master/README.md

Don't worry too much about getting the content right. Once the code is merged to our repository myself and others can edit it.

**Comment by ctrlaltvikas** [comment]
@peter-evans  I have kept most of the items of importance on the website. Small things are left. Just have a look at the content governance.surge.sh

**Comment by peter-evans** [comment]
@vikaskyadav It looks like it still shows all the default content. Am I missing something?

**Comment by ctrlaltvikas** [comment]
oh sorry. I forgot to push changes to surge. Have made changes live now.

**Comment by peter-evans** [comment]
@vikaskyadav Looks good. I think this is a good start for the website now. Please send a PR and we'll get the code merged and setup to publish from our repository. Thanks!

**Comment by peter-evans** [comment]
@vikaskyadav I've just noticed you have committed the code to a fork of the governance repository here:
https://github.com/vikaskyadav/governance/

Please don't create a PR to the governance repository. As requested in the task, please fork the following "web" repository and create a PR to merge there.
https://github.com/AvocadoNetwork/web

If for some reason you can't do the above let me know and I will just copy the code manually.

**Comment by ctrlaltvikas** [comment]
yeah. i noticed it. will do that asap.

**Comment by ctrlaltvikas** [comment]
@peter-evans  Could you please enable permissions for my account vikasharry03@gmail.com on this repo https://github.com/AvocadoNetwork/web. have moved the code. just need to push the code and everything will be at place.

**Comment by peter-evans** [comment]
@vikaskyadav I think you are trying to push directly to the organisation's repository. That won't work. Please follow these steps:
1. Commit your code to the master branch of the fork you made here: https://github.com/vikaskyadav/web-1
2. Create a pull request by clicking the "New Pull Request" button. Request to merge from your master branch to our master branch. See the screenshot below.

![image](https://user-images.githubusercontent.com/18365890/41241323-910efa72-6dd7-11e8-96bb-6a21e2873c35.png)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__Work has been started__.


These users each claimed they can complete the work by 2 months, 1 week from now.
Please review their action plans below:


**1) [vikaskyadav](https://gitcoin.co/profile/vikaskyadav) has started work.**

We can work on Graphics together. will be needing help in that.  I have some ideas for  elegant looking landing pages.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475).

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. **Submitted** 4. Done 

<hr>

__Work for 80000.0 AVO  has been submitted by__:

 1. [@vikaskyadav](https://gitcoin.co/profile/vikaskyadav)



@peter-evans please take a look at the submitted work:
* [PR](https://github.com/AvocadoNetwork/web/pull/1) by @vikaskyadav

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
$26,150.31 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by peter-evans** [comment]
Merged the PR. Thank you!

**Comment by peter-evans** [comment]
I've accepted the work and sent the bounty. It seems to be taking a while to reflect on gitcoin though. Hopefully should complete soon.

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__The funding of 80000.0 AVO  attached to this issue has been approved & issued to @vikaskyadav.__  

 * Learn more at [on the Gitcoin Issue Details page](https://gitcoin.co/issue/AvocadoNetwork/governance/3/475)
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $25,731.04 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
